frontends/web/src/routes/account/utils.ts

```diff
@@ -15,7 +15,21 @@
  * limitations under the License.
  */
 
-import { CoinCode, ScriptType } from '../../api/account';
+import { CoinCode, ScriptType, IAccount } from '../../api/account';
+
+export function getAccount(accounts: IAccount[], accountCode: string | undefined): IAccount | undefined {
```

Comment:
accountCode should be mandatory, this requires making code not optional in info.tsx, see later comment.

Reply:
Done :+1: